examples/simulation_pytorch/main.py

```diff
@@ -11,6 +11,8 @@
 from dataset_utils import get_cifar_10, do_fl_partitioning, get_dataloader
 from utils import Net, train, test
 
+import os
+from src.ProfilingThread import UtilMonitor
```

Comment:
Does this work? It might need to change to something like `from flwr.profiling_thread import ...`